src/ngraph/op/constant.cpp

```diff
@@ -48,14 +48,18 @@ string to_cpp_string(T value)
 
 constexpr NodeTypeInfo op::Constant::type_info;
 
+op::Constant::Constant(const shared_ptr<runtime::HostTensor>& tensor)
+    : Constant(tensor->get_element_type(), tensor->get_shape())
+{
+    m_tensor = tensor;
```

Comment:
Should copy the data since we can't guarantee that tensor won't change its stored value